src/app/styles/empty-state.css

```diff
@@ -49,11 +48,11 @@
 
 .preloader-circle {
   display: inline-block;
-  height: 24px;
-  width: 24px;
-  border-radius: 24px;
+  height: 16px;
+  width: 16px;
+  border-radius: 16px;
   background: #18a0fb;
-  margin: 8px;
+  margin: 6px;
```

Comment:
Feels a bit odd to me to use 6px increments rather than 4px, I think I preferred 24px with the spacing of 8.

Reply:
Changed to 8.